lib/srv/discovery/discovery.go

```diff
@@ -1057,6 +1057,40 @@ func (s *Server) emitUsageEvents(events map[string]*usageeventsv1.ResourceCreate
 	return nil
 }
 
+func (s *Server) submitFetchersEvent(fetchers []common.Fetcher) {
+	// Some Matcher types have multiple fetchers.
+	// Eg, rds Matcher converts into `rds` and `aurora` Fetchers
+	// The Fetch Event is meant to represent the configured matcher type: rds
```

Comment:
i think you meant to finish this sentence 😅

Reply:
It's probably not the best sentence but what I meant is that some Types convert into multiple fetchers, but we only want to capture the type and not the fetcher.

Example
If you add `rds` in the matchers type:
```yaml
aws:
  - types: [1rds]
    regions: [us-east-1]
```
This starts two fetchers: one for RDS and another one for Aurora.
However, we want to emit just a single event for the type, so only `rds`.

I'll try to re-word it